source/vstd/std_specs/option.rs

```diff
@@ -188,4 +188,21 @@ pub fn ex_option_clone<T: Clone>(opt: &Option<T>) -> (res: Option<T>)
     opt.clone()
 }
 
+// unwrap_or
+#[verifier::inline]
+pub open spec fn spec_ok_or<T, E>(option: Option<T>, err: E) -> Result<T, E> {
+    match option {
+        Some(t) => Ok(t),
+        None => Err(err),
+    }
+}
+
+#[verifier::external_fn_specification]
```

Comment:
Would it make sense to add `#[verifier::when_used_as_spec(spec_ok_or)]` here?

Reply:
Yes, thanks for noticing the oversight!

Reply:
Done